x-pack/solutions/observability/plugins/apm/public/components/shared/charts/chart_container.tsx

```diff
@@ -43,18 +43,21 @@ function LoadingChartPlaceholder({ height }: { height: number }) {
         alignItems: 'center',
         justifyContent: 'center',
       }}
+      data-test-subj={id}
     >
       <EuiLoadingChart data-test-subj="loading" size={'xl'} />
     </div>
   );
 }
 
-function FailedChartPlaceholder({ height }: { height: number }) {
+function FailedChartPlaceholder({ height, id }: { height: number; id?: string }) {
   return (
-    <EuiText color="subdued" style={{ height }}>
-      {i18n.translate('xpack.apm.chart.error', {
-        defaultMessage: 'An error happened when trying to fetch data. Please try again',
-      })}
-    </EuiText>
+    <div data-test-subj={id}>
```

Comment:
Do we need this extra div with data-test-subj here when we already have one for the EuiText?